build.psm1

```diff
@@ -568,7 +568,7 @@ Fix steps:
         if ($ReleaseTagToUse) {
             $ReleaseVersion = $ReleaseTagToUse
         } else {
-            $ReleaseVersion = (Get-PSCommitId) -replace '^v'
+            $ReleaseVersion = (Get-PSCommitId -WarningAction SilentlyContinue) -replace '^v'
```

Comment:
This change does not seem related to this PR.

Reply:
Yes, I will revert it.